src/index.html

```diff
@@ -8,42 +8,45 @@
     <link rel="stylesheet" href="./styles.css">
   </head>
   <body>
-
+    <section class="user-info">
+      <h1>Hi, I'm John</h1>
```

Comment:
we should get rid of this temporary text and leave it empty for the DOM display of a random user's name